src/IssueViz.Security/IssueVizSecurityPackage.cs

```diff
@@ -47,8 +48,16 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
         {
             await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
-            await HotspotsToolWindowCommand.InitializeAsync(this);
-            await TaintToolWindowCommand.InitializeAsync(this);
+            // We're not storing references to the command handler instances.
+            // We relying on the fact that the command handler instance registers a
+            // callback with the menu service to stop it from being garbage collected.
```

Comment:
I'm slightly worried about this, but logically it makes senses; the instances are reachable so they're not eligible for collection.